### PR TITLE
updated documentation for quickstarters and slaves

### PR DESCRIPTION
**Pull request by rattermeyer**
updated documentation on quickstarters and slaves

### CODE REVIEW

File: docs/modules/jenkins-slaves/pages/maven.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+= Maven / Gradle Slave
+
+This slave is used to build java code, both thru `maven` and `gradle`
+
+== Features
+
+* Nexus configuration for `Maven` and `Gradle`
+* HTTP Proxy awareness for `Maven` and `Gradle`
+* Default Installation maven 3.3 based on https://www.softwarecollections.org/en/scls/rhscl/rh-maven33/[rh-maven33]
+* Default Gradle Installation 5.6.1
```
Comment (michaelsauter) [inline docs/modules/jenkins-slaves/pages/maven.adoc]: I think it is way older?

Comment (rattermeyer) [reply docs/modules/jenkins-slaves/pages/maven.adoc]: you are right. I upgraded it for our installation. will provide PR for that one.

**Comment by clemensutschig** [comment]
@rattermeyer .. Status? Either Update and Merge or close

**Comment by rattermeyer** [comment]
Will close for now, because we needed to make more changes in the way we handle Java discovery to get it working on CentOS. Will open a seperate request for all changes.

**Pull request closed without merging**